cpp/benchmarks/fixture/benchmark_fixture.hpp

```diff
@@ -33,7 +34,8 @@ inline auto make_pool_instance()
 {
   static rmm::mr::cuda_memory_resource cuda_mr;
   static auto pool_mr =
-    std::make_shared<rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource>>(&cuda_mr);
+    std::make_shared<rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource>>(
+      &cuda_mr, rmm::percent_of_free_device_memory(50));
```

Comment:
I'm thinking about whether we should parse `percent_of_free_device_memory` from some env variable, instead of hard coding like this in many places. By doing so we can control the number we want at runtime.

Reply:
This PR is just fixing cuDF so that it has the same behavior, and doesn't break when I merge upcoming deprecation PRs. I will file a cuDF issue for this request, but I think the cuDF team should handle it.